recipes/jupyter-server-proxy/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "jupyter-server-proxy" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5a7d5bd044e43b40ca04df4040dd92efa66f7a98406e7b7b4c985fd86b18cbdd
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+    - setuptools
+  run:
+    - python >=3
+    - notebook
+    - simpervisor >=0.2
+    - aiohttp
+
+test:
+  imports:
+    - jupyter_server_proxy
+
+about:
+  home: https://github.com/jupyterhub/jupyter-server-proxy
+  license: BSD-3-Clause
+  # Vendoring the license file until the packages include it.
+  # xref: https://github.com/jupyterhub/jupyter-server-proxy/pull/105
+  license_file: LICENSE
```

Comment:
Submitted PR ( https://github.com/jupyterhub/jupyter-server-proxy/pull/105 ) to package the license file in PyPI packages. Have vendored it in the recipe for now.